src/gcp_scanner/scanner.py

```diff
@@ -26,12 +25,14 @@
 
 from . import crawl
 from . import credsdb
+from . import argparser
 from google.cloud import container_v1
 from google.cloud import iam_credentials
 from google.cloud.iam_credentials_v1.services.iam_credentials.client import IAMCredentialsClient
 from googleapiclient import discovery
 from httplib2 import Credentials
-from .models import SpiderContext
+from models import SpiderContext
```

Comment:
I don't think you changed SpiderContext, why is this changed?

Reply:
Corrected.